easybuild/easyconfigs/l/LocARNA/LocARNA-1.9.2-foss-2016b.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'LocARNA'
+version = '1.9.2'
+
+homepage = 'http://www.bioinf.uni-freiburg.de/Software/%(name)s/'
+description = """LocARNA is a collection of alignment tools for the structural analysis of RNA.
+ Given a set of RNA sequences, LocARNA simultaneously aligns and predicts common structures for
+ your RNAs. In this way, LocARNA performs Sankoff-like alignment and is in particular suited for
+ analyzing sets of related RNAs without known common structure."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/s-will/%(name)s/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['14aabc5425f5090bf1bcaa61c2db41f659e55dd40100db117fb9e40470b4a2b5']
+
+configopts = '--with-vrna=$EBROOTVIENNARNA'
```

Comment:
style nitpicking: please move this down, below `dependencies`

Reply:
@boegel I changed this.